e2e-tests/web/test/specs/navigateToDashboard.e2e.ts

```diff
@@ -0,0 +1,34 @@
+import LandingPage from '../pageobjects/landing.page';
+
+describe('Landing Page', () => {
+    it('Filter and Navigate to City', async () => {
+        // given
+        const language = 'en'
+        const filter = 'Augsburg'
+        const dashboardPath = `/${filter.toLowerCase()}/en`
+        await browser.setTimeout( {'implicit': 80000})
```

Comment:
```suggestion
        await browser.setTimeout({ 'implicit': 80000 })
```

Reply:
Isn't there already a timeout in `wdio.conf.js`?

Reply:
not for the implicit wait. it seems that the timeout is not set when you add this timeout to the capabilities